python/wtf-python/is-not-is-not.yml

```diff
@@ -0,0 +1,7 @@
+rules:
+  - id: is-not-is-not
+    patterns:
+      - pattern: $S is (not ...)
+    message: "is (not ...) probably not what you want to do"
```

Comment:
The description in the commit PR is so good! Maybe you could add that explanation here? 😄

Reply:
I second this :)